src/main/front/components/Modal/Signup/components/SignupForm.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+
+class SignupForm extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			stepCtr: 1,
+			maxStep: 1,
+			lastStep: 3
+		}
+
+		this.getCurrStep = this.getCurrStep.bind(this);
+		this.setStep = this.setStep.bind(this);
+		this.getPrevStep = this.getPrevStep.bind(this);
+		this.getNextStep = this.getNextStep.bind(this);
+		this.resendVerification = this.resendVerification.bind(this);
+	}
+
+	getCurrStep(step) {
```

Comment:
`getCurrStep = (step) => { rest of code };`
Follow this syntax for other functions (except lifecycle ones) and then remove their bindings in the contructor